conf/defaults.ini

```diff
@@ -435,6 +435,12 @@ user_invite_max_lifetime_duration = 24h
 # Enter a comma-separated list of usernames to hide them in the Grafana UI. These users are shown to Grafana admins and to themselves.
 hidden_users =
 
+[service_accounts]
+# Service account maximum expiration date in days.
+# Warning: When set, Grafana reduces the lifespan of _all existing tokens_ that have a days-to-live value greater than `token_expiration_day_limit` in order to comply with this setting.
+# This action is irreversible; you can't extend existing tokens' days-to-live by changing this setting to a larger value.
+token_expiration_day_limit = -1
```

Comment:
The default being empty would be more in line with existing optional settings

Reply:
Changed!